packages/core/admin/ee/admin/pages/Roles/CreatePage/index.js

```diff
@@ -39,31 +55,6 @@ const CreatePage = () => {
   const { isLoading: isLayoutLoading, data: permissionsLayout } = useFetchPermissionsLayout();
```

Comment:
We are missing the loading state in this view, can you check with @maevalienard what we want in terms of loading state for this view?

Reply:
Will do !